pytest_fixtures/api_fixtures.py

```diff
@@ -595,10 +584,10 @@ def module_lce_library(module_org):
 def module_puppet_classes(module_env_search):
     """Returns puppet class based on following criteria:
     Puppet environment from module_env_search and puppet class name. The name was set inside
-    module_cv_with_puppet_module.
+    module_import_puppet_module.
     """
     return entities.PuppetClass().search(
-        query={'search': f'name ~ {"generic_1"} and environment = {module_env_search.name}'}
+        query={'search': f'name ~ {"generic_1"} ' f'and environment = {module_env_search.name}'}
```

Comment:
This might have gotten adjusted by one of the pre-commit tools, there's an extra `f` string in here.

Also, `{"generic_1"}` doesn't really make sense being in `{}` within the f-string, as its just a string, there's no need for interpolation.

Reply:
this is forgotten change from testing, and meanwhile formatted by pre-commit, good eye. thanks :) 
sure, no need for f'string before variable module_env_search, brackets needs to stay -> query formatting

Reply:
If you want brackets in an f-string, you should double them so that they're interpreted as such.

https://docs.python.org/3/library/string.html#format-string-syntax

`If you need to include a brace character in the literal text, it can be escaped by doubling: {{ and }}.`